controller/src/main/java/org/jboss/as/controller/AbstractAttributeDefinitionBuilder.java

```diff
@@ -661,6 +662,19 @@ public BUILDER setCapabilityReference(CapabilityReferenceRecorder referenceRecor
         return (BUILDER)this;
     }
 
+    /**
+     * Sets a {@link AttributeDefinition#getUndefinedMetricValue()  default value} to use for the
+     * metric/runtime-attribute if no runtime value is available (e.g. we are a server running in admin-only mode.
```

Comment:
We aren't using this for non-metrics at this point, so I don't think the javadoc so refer to anything but metrics. If we want to use it for non-metrics we should change the method name, e.g. getUndefinedRuntimeValue() or something.